api/express.ts

```diff
@@ -51,10 +51,10 @@ app.use(
 );
 
 app.use('/', routers.healthRouter);
+app.use('/', protectedRoute(['admin', 'hm', 'shr', 'adm'], { requireAllRoles: false }), routers.userRouter);
```

Comment:
This won't work. Take a look at https://github.com/bcgov/citz-imb-kc-express/issues/14 and then read the README

Reply:
@BradyMitch Yeah I took a look at that when I ran into this issue. I ended up giving each role access to every endpoint, as it turns out most roles need access anyways. I know the solution was to add "route1", "route2", but I wasn't sure how that would be effected by the proxy considering we had to set it up with the '/'s in the first place. Let me know your thoughts. As it is now, it does seem to work :)